src/main/resources/data/po-lines/101101-1_pending_physical_checkin-true.json

```diff
@@ -46,7 +46,7 @@
       "percentage": 100.0
     }
   ],
-  "instanceId": "e3872ee8-91b1-4204-bc43-488f6821bc7b",
+  "instanceId": "c1d3be12-ecec-4fab-9237-baf728575185",
```

Comment:
As far as I can see you use instance ids from inventory reference data. Could you please also use following values (which are used by mod-orders in [InventoryHelper.java](https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/rest/impl/InventoryHelper.java#L460) to create instance record for consistency):

Instance | PO Line
--- | ---
source | source.code
title | title
editions[0] | edition
publication.publisher | publisher
publication.dateOfPublication | publicationDate
contributors[].name | contributors[].contributor
identifiers[].value | details.productIds[].productId